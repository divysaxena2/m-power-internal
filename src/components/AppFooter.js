import React, { Component } from 'react';
import getTheme from '../../native-base-theme/components'
import material from '../../native-base-theme/variables/material';
import { Image, StyleSheet } from 'react-native';
import { Button, StyleProvider, Icon, Footer, Text, FooterTab } from 'native-base';

type Props = {};
export default class AppFooter extends Component<Props> {
    render() {
        return (
            <StyleProvider style={getTheme(material)}>
                <Footer>
                    <FooterTab>
                        <Button>
                            <Image source={require('../assets/ribbon.png')} style={{ height: 34, width: 38 }} />
                            <Text uppercase={false} style={styles.captions}>Home</Text>
                        </Button>
                        <Button>
                            <Image source={require('../assets/ribbon.png')} style={{ height: 34, width: 38 }} />
                            <Text uppercase={false} style={styles.captions}>Services</Text>
                        </Button>
                        <Button>
                            <Image source={require('../assets/ribbon.png')} style={{ height: 34, width: 38 }} />
                            <Text uppercase={false} style={styles.captions}>Circles</Text>
                        </Button>
                        <Button >
                            <Image source={require('../assets/ribbon.png')} style={{ height: 34, width: 38 }} />
                            <Text uppercase={false} style={styles.captions}>Utilities</Text>
                        </Button>
                    </FooterTab>
                </Footer>
            </StyleProvider>)
    }
}
const styles = StyleSheet.create({
    captions: {
        color: '#003A64',
        fontSize: 13,
        letterSpacing: 0.62,
        lineHeight: 18,
        height: 18,
        justifyContent: 'center',
        alignItems: 'center'

    }
});