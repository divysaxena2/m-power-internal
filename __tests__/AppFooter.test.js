import 'react-native';
import React from 'react';
import AppFooter from '../src/components/AppFooter';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer.create(
        <AppFooter />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});