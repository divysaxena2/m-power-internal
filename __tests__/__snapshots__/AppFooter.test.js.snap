// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FEFFFF",
      "borderColor": undefined,
      "borderTopWidth": undefined,
      "elevation": 3,
      "flexDirection": "row",
      "height": 56,
      "justifyContent": "center",
      "left": 0,
      "paddingBottom": 0,
      "right": 0,
    }
  }
>
  <View
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": undefined,
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "space-between",
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "alignSelf": "center",
          "backgroundColor": "transparent",
          "borderBottomWidth": null,
          "borderColor": null,
          "borderLeftWidth": null,
          "borderRadius": 2,
          "borderRightWidth": null,
          "borderTopWidth": null,
          "borderWidth": undefined,
          "elevation": 0,
          "flex": 1,
          "flexDirection": null,
          "height": 56,
          "justifyContent": "center",
          "opacity": 1,
          "paddingBottom": 6,
          "paddingTop": 6,
          "shadowColor": null,
          "shadowOffset": null,
          "shadowOpacity": null,
          "shadowRadius": null,
        }
      }
    >
      <Image
        source={1}
        style={
          Object {
            "height": 34,
            "width": 38,
          }
        }
      />
      <Text
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "color": "#bfc6ea",
              "fontFamily": "Roboto",
              "fontSize": 11,
              "lineHeight": 16,
              "marginLeft": 0,
              "marginRight": 0,
              "paddingLeft": 16,
              "paddingRight": 16,
            },
            Object {
              "alignItems": "center",
              "color": "#003A64",
              "fontSize": 13,
              "height": 18,
              "justifyContent": "center",
              "letterSpacing": 0.62,
              "lineHeight": 18,
            },
          ]
        }
        uppercase={false}
      >
        Home
      </Text>
    </View>
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "alignSelf": "center",
          "backgroundColor": "transparent",
          "borderBottomWidth": null,
          "borderColor": null,
          "borderLeftWidth": null,
          "borderRadius": 2,
          "borderRightWidth": null,
          "borderTopWidth": null,
          "borderWidth": undefined,
          "elevation": 0,
          "flex": 1,
          "flexDirection": null,
          "height": 56,
          "justifyContent": "center",
          "opacity": 1,
          "paddingBottom": 6,
          "paddingTop": 6,
          "shadowColor": null,
          "shadowOffset": null,
          "shadowOpacity": null,
          "shadowRadius": null,
        }
      }
    >
      <Image
        source={1}
        style={
          Object {
            "height": 34,
            "width": 38,
          }
        }
      />
      <Text
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "color": "#bfc6ea",
              "fontFamily": "Roboto",
              "fontSize": 11,
              "lineHeight": 16,
              "marginLeft": 0,
              "marginRight": 0,
              "paddingLeft": 16,
              "paddingRight": 16,
            },
            Object {
              "alignItems": "center",
              "color": "#003A64",
              "fontSize": 13,
              "height": 18,
              "justifyContent": "center",
              "letterSpacing": 0.62,
              "lineHeight": 18,
            },
          ]
        }
        uppercase={false}
      >
        Services
      </Text>
    </View>
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "alignSelf": "center",
          "backgroundColor": "transparent",
          "borderBottomWidth": null,
          "borderColor": null,
          "borderLeftWidth": null,
          "borderRadius": 2,
          "borderRightWidth": null,
          "borderTopWidth": null,
          "borderWidth": undefined,
          "elevation": 0,
          "flex": 1,
          "flexDirection": null,
          "height": 56,
          "justifyContent": "center",
          "opacity": 1,
          "paddingBottom": 6,
          "paddingTop": 6,
          "shadowColor": null,
          "shadowOffset": null,
          "shadowOpacity": null,
          "shadowRadius": null,
        }
      }
    >
      <Image
        source={1}
        style={
          Object {
            "height": 34,
            "width": 38,
          }
        }
      />
      <Text
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "color": "#bfc6ea",
              "fontFamily": "Roboto",
              "fontSize": 11,
              "lineHeight": 16,
              "marginLeft": 0,
              "marginRight": 0,
              "paddingLeft": 16,
              "paddingRight": 16,
            },
            Object {
              "alignItems": "center",
              "color": "#003A64",
              "fontSize": 13,
              "height": 18,
              "justifyContent": "center",
              "letterSpacing": 0.62,
              "lineHeight": 18,
            },
          ]
        }
        uppercase={false}
      >
        Circles
      </Text>
    </View>
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "alignSelf": "center",
          "backgroundColor": "transparent",
          "borderBottomWidth": null,
          "borderColor": null,
          "borderLeftWidth": null,
          "borderRadius": 2,
          "borderRightWidth": null,
          "borderTopWidth": null,
          "borderWidth": undefined,
          "elevation": 0,
          "flex": 1,
          "flexDirection": null,
          "height": 56,
          "justifyContent": "center",
          "opacity": 1,
          "paddingBottom": 6,
          "paddingTop": 6,
          "shadowColor": null,
          "shadowOffset": null,
          "shadowOpacity": null,
          "shadowRadius": null,
        }
      }
    >
      <Image
        source={1}
        style={
          Object {
            "height": 34,
            "width": 38,
          }
        }
      />
      <Text
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "color": "#bfc6ea",
              "fontFamily": "Roboto",
              "fontSize": 11,
              "lineHeight": 16,
              "marginLeft": 0,
              "marginRight": 0,
              "paddingLeft": 16,
              "paddingRight": 16,
            },
            Object {
              "alignItems": "center",
              "color": "#003A64",
              "fontSize": 13,
              "height": 18,
              "justifyContent": "center",
              "letterSpacing": 0.62,
              "lineHeight": 18,
            },
          ]
        }
        uppercase={false}
      >
        Utilities
      </Text>
    </View>
  </View>
</View>
`;
